prow/config.yaml

```diff
@@ -3275,7 +3275,6 @@ presubmits:
         imagePullPolicy: Always
         args:
         - "--clean"
-        - "--git-cache=/root/.cache/git"
```

Comment:
(this is in `pull-security-kubernetes-verify-prow`)

Reply:
fixed

Reply:
and assume we don't need to mount the volume in the job anymore

Reply:
the volume is used for docker image caching :-)

Reply:
discussed out of band, updated volumes a bit more.